### ISSUE TITLE
Should set DEBUG=False?

### ISSUE THREAD

**Issue by megies** [krischer/jane#38]
Django docs (and also doing a `python3 manage.py check --deploy`) mention that `DEBUG=True` should never be used in production but it's set in `settings.py` and docs don't mention changing this.

Should we manually set `DEBUG` to `False`?

**Comment by barsch** [comment]
yes this should be ```False``` by default - can be changed directly in master

**Comment by megies** [comment]
I'm getting a "Server Error" page when trying to log into the admin interface and when `DEBUG=False`..
